README.md

```diff
@@ -15,26 +15,25 @@ A general-purpose Kubernetes [admission webhook](https://kubernetes.io/docs/refe
 
 Apart from webhook validation, this tool can also do [one-time outside of cluster scan](#on-demand-outside-of-cluster-scan) for objects that are violating validation rules.
 
-Can be set up to validate that:
-* containers have their resource limits specified (`memory`, `cpu`)
-* containers have their resource requests specified (`memory`, `cpu`)
-* containers have readonly root filesystem
-* ingress rules hosts and paths match regex
-* ingress rules hosts and paths are not in collision with definitions already in the cluster
-* ingress tls hosts match regex
-* ingress tls hosts are not in collision with definitions already in the cluster
-
-Resource spec validation operates on:
-* `Pod`s
-* `Deployment`s
-* `ReplicaSet`s
-* `DaemonSet`s
-* `Job`s
-* `CronJob`s
-* `StatefulSet`s
-
-Host and path validation operates on:
-* `Ingress`es
+Validation features:
+* Container setup validation
+    * Containers have their resource limits specified (`memory`, `cpu`)
+    * Containers have their resource requests specified (`memory`, `cpu`)
+    * Containers have readonly root filesystem
+* Ingress setup validation
+    * Ingress rules hosts and paths match regex
+    * Ingress rules hosts and paths are not in collision with definitions already in the cluster
+    * Ingress tls hosts match regex
+    * Ingress tls hosts are not in collision with definitions already in the cluster
+* Annotation validation
+    * Annotations match give rule set (See [annotation-rules-example.yaml](annotation-rules-example.yaml))
```

Comment:
Please add comment that annotation exceptions are treated as prefixes. Maybe I am blind but I have not seen any mention about that, just saw that in the code and I was a bit surprised :)

I also suggest to add some prefix examples in the example annotation rule yaml. Somehting like "nasty-annocation-group-" and "allowed-annotation-group-".